help/sources/tutorials/api/create/marketing-automation/pathfactory.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Create a PathFactory Base Connection Using the Flow Service API
+description: Learn how to authenticate your PathFactory account against Experience Platform using the Flow Service API.
+exl-id: 
```

Comment:
We can remove exl-id here, but let's add badge: Beta to the metadata here as well

Reply:
Fixed